bindings/java/README.md

```diff
@@ -25,7 +25,7 @@ the [Makefile](./Makefile).
 make test
 ```
 
-## Library
+## Public Maven Repo
 
 The library which uses this binding and publishes a package to a public maven repo
 is [jc-kzg-4844](https://github.com/ConsenSys/jc-kzg-4844).
```

Comment:
This reminds me, we should add this link to the readme too:

https://central.sonatype.com/artifact/io.consensys.protocols/jc-kzg-4844